### PR TITLE
PT-2917: Sections and fields should be marked as disabled in patient/family forms in a consistent manner

### PR DESCRIPTION
**Pull request by marta-**

### CODE REVIEW

File: components/family-studies/ui/src/main/resources/PhenoTips/FamilySheet.xml [hunk 1]
```diff
@@ -97,8 +97,11 @@
 ##
 ## Extensions displayed after the content
 #foreach ($e in $services.uix.getExtensions('phenotips.familyRecord.after', {'sortByParameter' : 'order'}))
+  #set ($extension_disabled = ($e.getParameters().get('enabled') == 'false'))
+  #if (!$extension_disabled)
+    {{html clean=false}}$services.rendering.render($e.execute(), 'xhtml/1.0'){{/html}}
```
Comment (marta-) [inline components/family-studies/ui/src/main/resources/PhenoTips/FamilySheet.xml]: Rather _unfortunately_. If we had had any, I would have noticed the problem. Thanks!

File: components/family-studies/ui/src/main/resources/PhenoTips/FamilySheet.xml [hunk 2]
```diff
@@ -97,33 +97,11 @@
...
-      #set ($pedigreeHtml = $services.rendering.render($e.execute(), 'xhtml/1.0'))
-    #end
-  #end
-  #if ("$!{pedigreeHtml}" != '')
-    (% id='pedigree' class='chapter clinical-info pedigree-family-section' %)(((
-      {{html wiki=false clean=false}}$pedigreeHtml{{/html}}
-    )))
-  #end
-{{/velocity}}{{/comment}}</content>
+#end</content>
```
Comment (sdumitriu) [inline components/family-studies/ui/src/main/resources/PhenoTips/FamilySheet.xml]: You forgot to close `{{/velocity}}`.

Comment (marta-) [inline components/family-studies/ui/src/main/resources/PhenoTips/FamilySheet.xml]: Technically, closing the macro at the end of the content is not mandatory. But yes, there should be a closing tag, thanks for noticing.

File: components/family-studies/ui/src/main/resources/PhenoTips/FamilySheet.xml [hunk 3]
```diff
@@ -97,8 +97,11 @@
 ##
```
Comment (sdumitriu) [inline components/family-studies/ui/src/main/resources/PhenoTips/FamilySheet.xml]: Wrong commit title, this is for PT-2922...

**Pull request merged**